app/Http/Controllers/UserController.php

```diff
@@ -75,7 +75,7 @@ public function store(Request $request){
     }
 
     public function show(Request $request, User $user){
-        if(Auth::user()->id != $user->id && !Auth::user()->hasRole(Config::get('constants.Administrator'),Config::get('constants.Certificate_administrator'))){
+        if(Auth::user()->id != $user->id && !Auth::user()->hasRole(Config::get('constants.Administrator'),Config::get('constants.Certificate_administrator'),Config::get('constants.NSAC_emergency_info_administrator'))){
```

Comment:
Het lijkt erop dat de rol hier toegang heeft tot het gehele User object. Wat stuurt de API terug naar de frontend? is het het gehele user object? 

of is het een deel van het user object e.g: 
{
   "name": "indie peeters"
   "emergencyInfo": {
       "emergencyPhonenumber": "phonenumber"
       "emergencyAddress": "adress"
       "emergencyCity": "city"
   }
}

Reply:
Oeh dat is waar ja, zoals ik het had gemaakt filtert de frontend de data maar word het idd wel allemaal gestuurd volgens mij. Ik kan kijken of ik het zo kan krijgen dat het in de API al gefilterd word

Reply:
Prima! laat maar weten als je hulp nodig hebt :)